salt/modules/opkg.py

```diff
@@ -997,7 +1059,18 @@ def list_pkgs(versions_as_list=False, **kwargs):
 
     cmd = ['opkg', 'list-installed']
     ret = {}
-    out = __salt__['cmd.run'](cmd, output_loglevel='trace', python_shell=False)
+    out_dict = __salt__['cmd.run_all'](cmd, output_loglevel='trace', python_shell=False)
+
+    if out_dict['retcode'] != 0:
+        if out_dict['stderr']:
+            log.warning('error error')
+            errors.append(out_dict['stderr'])
+        else:
+            log.warning('error stdout')
```

Comment:
I think you forget to remove this. Or do you want to keep but with a proper message?